test/CMakeLists.txt

```diff
@@ -1,26 +1,31 @@
+include(ExternalProject)
+
+ExternalProject_Add(
+	googletest_project
+	SOURCE_DIR "${CMAKE_SOURCE_DIR}/test/gtest-1.8.0"
+	INSTALL_DIR "${CMAKE_BINARY_DIR}/prefix"
+	CMAKE_ARGS -DCMAKE_INSTALL_PREFIX:PATH=<INSTALL_DIR> -DBUILD_GMOCK=ON
+)
+
+add_library(gmock UNKNOWN IMPORTED)
+set_target_properties(gmock PROPERTIES
+	IMPORTED_LOCATION ${PROJECT_BINARY_DIR}/prefix/lib/libgmock.a
+)
+
+find_package(Threads)
```

Comment:
What's this package?

Reply:
On Linux Googletest requires pthread library linking. Here we find this library (optional, since for Windows we don't need them), and next ${CMAKE_THREAD_LIBS_INIT} is added as -lpthread linker option.